src/Illuminate/Queue/TimeoutExceededException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Illuminate\Queue;
+
+use RuntimeException;
+
+class TimeoutExceededException extends RuntimeException
```

Comment:
This could be sent to 10.x if you extended MaxAttemptsExceededException instead of RuntimeException.